libpod/container_api.go

```diff
@@ -74,7 +65,8 @@ func (c *Container) Init(ctx context.Context) (err error) {
 // started
 // Stopped containers will be deleted and re-created in runc, undergoing a fresh
 // Init()
-func (c *Container) Start(ctx context.Context) (err error) {
+// if recursive is set, Start will also start all containers this ctr depends on
```

Comment:
Can you add periods at the end of all the sentences in this comment? I think godoc will mangle the newlines, so we need periods to preserve sentence flow.

Also, can you capitalize the `if` at the beginning here?